DEVELOPER_SETUP.md

```diff
@@ -0,0 +1,415 @@
+
+Once everything is running successfully:
+
+| Service | URL | Credentials | Notes |
+|---------|-----|-------------|-------|
+| **Backend Admin** | http://localhost:10645 | admin/store | Admin dashboard |
+| **Frontend Store** | https://localhost:3000 | N/A | Accept certificate warning |
+| **SQL Server** | localhost:1433 | sa/Pass@word | If using Docker |
+
+## 🔍 Verification Steps
+
+1. **Backend Check:**
+   ```bash
+   # Should return 200 OK
+   curl -I http://localhost:10645
+   ```
+
+2. **Frontend Check:**
+   ```bash
+   # Check processes are running
+   Get-Process node -ErrorAction SilentlyContinue
+   
+   # Check port is listening
+   netstat -an | findstr ":3000" | findstr "LISTENING"
+   ```
+
+3. **Database Check:**
+   - Login to admin panel (http://localhost:10645)
+   - Navigate to Settings → General
+   - Should see platform settings loaded
+
+## 📚 Additional Resources
+
+- [Full Documentation](https://docs.virtocommerce.org)
+- [Developer Guide](https://docs.virtocommerce.org/platform/developer-guide/)
+- [Module Development](https://docs.virtocommerce.org/platform/developer-guide/create-new-module/)
+- [Frontend Documentation](https://docs.virtocommerce.org/storefront/)
+
+## 🆘 Getting Help
+
+If you encounter issues not covered here:
+
+1. **Check GitHub Issues**: [Platform Issues](https://github.com/VirtoCommerce/vc-platform/issues)
+2. **Community Forum**: [virtocommerce.org](https://www.virtocommerce.org)
+3. **Documentation**: [docs.virtocommerce.org](https://docs.virtocommerce.org)
+4. **Create New Issue**: Include error messages, OS version, and steps to reproduce
+
+## 🤖 **MCP Server Module Installation (Optional)**
+
+The [VirtoCommerce MCP Server module](https://github.com/VirtoCommerce/vc-module-mcp-server) enables AI agents like Claude Desktop to interact with VirtoCommerce APIs.
+
+### **Prerequisites for MCP Server**
+- VirtoCommerce Platform running successfully
+- Node.js 18+ (for mcp-remote proxy)
+- Claude Desktop (optional, for testing)
+
+### **Installation Steps**
+
+1. **Clone and build the MCP Server module:**
+```bash
+# In D:\work directory
+git clone https://github.com/VirtoCommerce/vc-module-mcp-server.git
+cd vc-module-mcp-server
+dotnet build VirtoCommerce.McpServer.sln
+```
+
+2. **Deploy to VirtoCommerce:**
+```bash
+# Create module directory
+New-Item -ItemType Directory -Path "D:\work\vc-platform\modules\VirtoCommerce.McpServer" -Force
+
+# Copy built files
+Copy-Item "src\VirtoCommerce.McpServer.Web\bin\Debug\net8.0\*" "D:\work\vc-platform\modules\VirtoCommerce.McpServer" -Recurse -Force
+
+# Copy essential module files
+Copy-Item "src\VirtoCommerce.McpServer.Web\module.manifest" "D:\work\vc-platform\modules\VirtoCommerce.McpServer\" -Force
+New-Item -ItemType Directory -Path "D:\work\vc-platform\modules\VirtoCommerce.McpServer\Scripts" -Force
+Copy-Item "src\VirtoCommerce.McpServer.Web\Scripts\*" "D:\work\vc-platform\modules\VirtoCommerce.McpServer\Scripts\" -Recurse -Force
```

Comment:
This is too complicated. A better way is cloning right into vc-platform\modules or creating a link with `mklink`